tutorials/hana-cloud-dl-clients-overview/hana-cloud-dl-clients-overview.md

```diff
@@ -314,33 +335,35 @@ For additional details on the SAP HANA database explorer, see the tutorial [Get
 
     ![GUI Installer](windows-gui-install.png)
 
-    Console mode installer
+    Console mode installer on Linux
 
     ![Product Features](productfeatures.png)
 
     Follow the remaining prompts to finish the installation.
 
-4.  The installation location can be referenced after the installation is complete by the following environment variable.
+4.  The installation location can be referenced through an environment variable.
+
+  * On Microsoft Windows, in a newly opened command prompt run the following to see the installation location.
 
     ```Shell (Windows)
     ECHO %IQDIR17%
     ```
 
-    >Note that this tutorial uses notepad and `pico` as default text editors, but any text editor will do.
-    >`pico` can be installed on SUSE Linux with
-
-    >```Shell (Linux SUSE)
-    sudo zypper install pico
-    >```
-
-    On Linux, this environment variable and others are set in a file named `IQ.sh`.  Configure it to be run each time the Bash shell is started by referencing it in `.bash_profile` or possibly `.bashrc`.
+  * On Linux, this environment variable and others are set in a file named `IQ.sh`.  Configure it to be run each time the Bash shell is started by referencing it in `.bash_profile` or possibly `.bashrc`.
 
     Open the `.bash_profile`.
 
     ```Shell (Linux)
     pico ~/.bash_profile
     ```
 
+    >Note that this tutorial uses notepad and `pico` as default text editors, but any text editor will do.

```

Comment:
Please rephrase and do not start a note with the keyword "Note", but use a label instead, for example, \*\*Important:\*\*